app/components/public/ticket-list.js

```diff
@@ -1,20 +1,27 @@
 import Ember from 'ember';
 import FormMixin from 'open-event-frontend/mixins/form';
-import { sumBy } from 'lodash';
 
 const { Component, computed } = Ember;
 
 export default Component.extend(FormMixin, {
   classNames: ['ui', 'segments', 'ticket-list'],
 
   hasTicketsInOrder: computed('tickets.@each.orderQuantity', function() {
-    return sumBy(this.get('tickets'), 'orderQuantity') > 0;
+    let totalQuantity = 0.0;
+    this.get('tickets').forEach(ticket => {
+      if (ticket.orderQuantity) {
+        totalQuantity += ticket.orderQuantity;
+      }
+    });
+    return totalQuantity > 0;
   }),
 
   total: computed('tickets.@each.orderQuantity', function() {
     let sum = 0.0;
     this.get('tickets').forEach(ticket => {
```

Comment:
This could be written as 

```js
return sumBy(
  this.get('tickets'), 
  ticket => ticket.get('price') * ticket.getWithDefault('orderQuantity', 0)
);
```